packages/bits/src/lib/repeat/repeat.component.less

```diff
@@ -64,7 +70,7 @@
             }
         }
 
-        &__disabled {
+        &--disabled {
```

Comment:
lol, shouldn't we let the old one too? &__disabled, &--disabled { ...

Reply:
I just wanted to unify the names (other classes use --), it's internal for the component so shouldn't be an issue